uint8 NONE = 0
uint8 LANDMARK = 1

uint8 type
uint64 key
geometry_msgs/Point point